doc/bitcoin-conf.md

```diff
@@ -0,0 +1,11 @@
+# `bitcoin.conf` - Bitcoin configuration file
+
+All command-line options (except for `-conf`) may be specified in a configuration file, and all configuration file options (except for `-includeconf`) may also be specified on the command line. Command-line options override values set in the configuration file and configuration file options override values set in the GUI.
+
+The configuration file is a plain text file and consist of 'setting=value' entries, one per line. Leading and trailing whitespaces are removed. A number sign (`#`) indicates a comment. Comments and empty lines are ignored.
```

Comment:
Could mention that, although possible, it's not recommended to have a setting and a comment in the same line?
```
foo=bar # reasons
```
vs
```
# reasons
foo=bar
```